src/webgpu/api/validation/compute_pipeline.spec.ts

```diff
@@ -283,7 +283,7 @@ Tests calling createComputePipeline(Async) validation for overridable constants
         { constants: { 9999: 0 }, _success: false },
         { constants: { 1000: 0, c2: 0 }, _success: false },
         { constants: { 数: 0 }, _success: true },
-        { constants: { séquençage: 0 }, _success: true }, // test unicode normalization
+        { constants: { séquençage: 0 }, _success: false }, // test unicode normalization
```

Comment:
So this tests that no unicode normalization happens because both sequencages are different ways to write it? If that's the case maybe the comment could be expanded slightly to explain that.

Reply:
updated